examples/mini-hlint/mini-hlint.cabal

```diff
@@ -9,10 +9,12 @@ copyright:           Digital Asset 2018-2019
 build-type:          Simple
 cabal-version:       >=1.10
 
+
 executable mini-hlint
   main-is:             Main.hs
   build-depends:       base >=4.11
                      , extra
                      , ghc-lib-parser
   default-language:    Haskell2010
   hs-source-dirs:      src
+  ghc-options: -Wmissing-exported-signatures
```

Comment:
I generally don't put ghc warnings options in the .cabal file. Warnings are for my benefit, while the Cabal is for distribution. Instead I put these warning checks in the .ghci file and in the CI.

Reply:
That was a mistake - detritus left behind from some debugging. Will fix.